include/onnxruntime/core/session/onnxruntime_session_options_config_keys.h

```diff
@@ -93,6 +93,15 @@ static const char* const kOrtSessionOptionsMemoryOptimizerEnabler = "optimizatio
 static const char* const kOrtSessionOptionsMemoryOptimizerProbeConfig = "optimization.enable_memory_probe_recompute_config";
 #endif
 
+// This setting if set should contain a comma separated list of optimizers names that should be disabled.
+// Optimizers may take time to execute and affect model loading time. If you feel that a specific optimizer
+// does not provider runtime benefits, but affects your model loading time you may disable it using this config
+// entry. This option is not enabled in ORT_MINIMAL_BUILD build.
+// A list of optimizes is available in onnxruntime/core/optimizer/graph_transformer_utils.cc
+//
+// Default is an empty string which means no optimizers are disabled.
+static const char* const kOrtSessionOptionsDisableSpecifiedOptimizers = "optimization.disable_specified_optimizers";
```

Comment:
nit: most of the configs in this file that have names starting with "enable" or "disable" expect boolean values. May be we should rename this to "optimizers_to_disable" or simply "disabled_optimizers"?

Reply:
`optimization` is a widely used domain in the config. I would hesitate to put this setting into a league of its own.

Reply:
Also `enable/disable` are booleans, and this setting is not.

Reply:
Yes, I meant using something like `optimization.disabled_optimizers`.